src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
+  display: flex;
+  justify-content: center;
   margin: 0;
+  background: #fff;
+  --link-color: #00acdc ;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  width: 100%;
+  padding: 0 50px;
+}
+
+.header__logo {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: block;
+  padding: 23px 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav__link:hover {
```

Comment:
merge `nav__link:hover` and` is-active` styles cause as you can see they have the same colors